README.md

```diff
@@ -23,7 +23,7 @@ Remark42 is a self-hosted, lightweight and simple (yet functional) comment engin
 * Clean, lightweight and customizable UI with white and dark themes
 * Multi-site mode from a single instance
 * Integration with automatic SSL (direct and via [nginx-le](https://github.com/nginx-le/nginx-le))
-* [Privacy focused](#privacy)
+* [Privacy focused](https://remark42.com/#privacy)
```

Comment:
I'm not sure if it's right link

Reply:
yeah, this is the one https://remark42.com/privacy/

Reply:
It led to a particular part of the text I'm referencing before the change: it was moved to the site's main page from the Readme but was not appropriately referenced before.

Reply:
Commit: https://github.com/umputun/remark42/commit/19a4eada25fce21e73ac2bf310361c352f4f0683